config/v3_4_experimental/schema/ignition.json

```diff
@@ -141,7 +141,10 @@
             }
           }
         }
-      }
+      },
+      "required": [
```

Comment:
Would it make sense to backport this to previous schemas as well?

Reply:
That would change a field type (in non-Go bindings) in stable specs, so no.  Actually I suspect we should avoid even trivial changes to stable JSON schema to avoid confusion, since the schema may have been copied into other repos.